src/ck-core/ckcheckpoint.C

```diff
@@ -30,6 +30,8 @@ void noopit(const char*, ...)
 #define DEBUGC(x) x
 //#define DEBUGC(x) 
 
+#define PE_GROUP_SIZE 256
```

Comment:
I think this (and other instances where "group" is used in this way) should be changed to avoid confusion with Charm++ groups. Maybe "subdirectory" or something like that.